src/conn/conn_compact.c

```diff
@@ -402,6 +402,10 @@ __wt_background_compact_end(WT_SESSION_IMPL *session)
           session, background_compact_ema, conn->background_compact.bytes_rewritten_ema);
     }
 
+    __wt_verbose_info(session, WT_VERB_COMPACT_PROGRESS,
+      "%s: background compaction finished (status: %s) - reclaimed %" PRIu64 "bytes", uri,
```

Comment:
Missing space as seen in our testing:
```suggestion
      "%s: background compaction finished (status: %s) - reclaimed %" PRIu64 " bytes", uri,
```

Reply:
resolved 14df36244784480a3db5d7f68cfb7c551991809a